lib/ProductOpener/APIProductWrite.pm

```diff
@@ -186,13 +187,76 @@ sub update_packagings ($request_ref, $product_ref, $field, $is_addition, $value)
 	return;
 }
 
-=head2 update_product_fields ($request_ref, $product_ref)
+=head2 update_tags_fields ($request_ref, $product_ref, $field, $is_addition, $value)
+
+Update packagings.
+
+=cut
+
+sub update_tags_fields ($request_ref, $product_ref, $field, $is_addition, $tags_lc, $value) {
```

Comment:
I'm not sure $is_addition is a good name. It can mean, that the tags are new or that it's adding to existing tags…
I would better have a $do_replace (which would be the contrary of $is_addition ?).

Reply:
$add_to_existing_tags would be better I think, it's more in line with the API logic: we replace by default, or we add if the field ends with _add

Reply:
It mimicked what we had for packagings (adding a new packaging component):

update_packagings($request_ref, $product_ref, $field, $is_addition, $value)

Reply:
:+1: for `$add_to_existing_tags`